homeassistant/components/reisingerbridge/entity.py

```diff
@@ -0,0 +1,50 @@
+"""Entity for the reisinger drive component."""
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DOMAIN
+
+
+class OpenReisingerEntity(CoordinatorEntity):
+    """Representation of a OpenReisinger entity."""
+
+    def __init__(self, open_reisinger_data_coordinator, device_id, description=None):
+        """Initialize the entity."""
+        super().__init__(open_reisinger_data_coordinator)
+
+        if description is not None:
+            self.entity_description = description
+            self._attr_name = device_id
+            self._attr_unique_id = f"{device_id}_{description.key}"
+        else:
+            self._attr_unique_id = device_id
+            self._attr_name = device_id
+
+        self._device_id = device_id
+        self._update_attr()
```

Comment:
This should not be needed? As you are using a data update coordinator, the recent data is always available and updating isn't needed?